README.md

```diff
@@ -1,103 +1,13 @@
-Doctrine Coding Standard
-========================
+# Doctrine Coding Standard
 
 [![Build Status](https://img.shields.io/travis/doctrine/coding-standard/master.svg?style=flat-square)](http://travis-ci.org/doctrine/coding-standard)
 [![Total Downloads](https://img.shields.io/packagist/dt/doctrine/coding-standard.svg?style=flat-square)](https://packagist.org/packages/doctrine/coding-standard)
 [![Latest Stable Version](https://img.shields.io/packagist/v/doctrine/coding-standard.svg?style=flat-square)](https://packagist.org/packages/doctrine/coding-standard)
 
+The Doctrine Coding Standard is a set of PHPCS rules applied to all Doctrine projects.
```

Comment:
I think the full name with the link was better than abbreviation without the link.